src/sdkBase.ts

```diff
@@ -74,10 +74,6 @@ export abstract class SdkBase {
     this.initInternal(opts);
   }
 
-  public isInitialized() {
-    return this._initComplete;
-  }
-
```

Comment:
Don't need duplicate functions

Reply:
I don't know the history of this method, but typically deleting a public function here scares me a bit because is a breaking change - participants could be relying on this method. 

Are we happy to classify the next release as a major version change or are we certain this method is never used? Or something else I am missing.

Reply:
This is just a function that I wrote on my last branch - I don't think its deployed at all.  Scott and I didn't realize the other person was writing one of these functions, so we ended up with two different functions returning the same variable.  Should be good to delete